source/en/sdr.rst

```diff
@@ -465,39 +465,39 @@ Interpreting Results
 
-    * **usle.tif** (type: raster; units: tons/pixel/year): Total potential soil loss per pixel in the original land cover calculated from the USLE equation. (Eq. :eq:`usle`)
+    * **usle.tif** (type: raster; units: metric tons/hectare/year): Total potential soil loss in the original land cover calculated from the USLE equation. (Eq. :eq:`usle`)
 
-    * **avoided_erosion.tif** (type: raster; units: tons/pixel/year): The contribution of vegetation to keeping soil from eroding from each pixel. (Eq. :eq:`aer_i`)
+    * **avoided_erosion.tif** (type: raster; units: metric tons/hectare/year): The contribution of vegetation to keeping soil from eroding from each pixel. (Eq. :eq:`aer_i`)
 
-    * **avoided_export.tif** (type: raster; units: tons/pixel/year): The contribution of vegetation to keeping erosion from entering a stream. This combines local/on-pixel sediment retention with trapping of erosion from upslope of the pixel.  (Eq. :eq:`aex_i`)
+    * **avoided_export.tif** (type: raster; units: metric tons/hectare/year): The contribution of vegetation to keeping erosion from entering a stream. This combines local/on-pixel sediment retention with trapping of erosion from upslope of the pixel.  (Eq. :eq:`aex_i`)
 
     * **watershed_results_sdr.shp**: Table containing biophysical values for each watershed, with fields as follows:
 
-        * **sed_export** (units: tons/watershed/year): Total amount of sediment exported to the stream per watershed. This should be compared to any observed sediment loading at the outlet of the watershed. Knowledge of the hydrologic regime in the watershed and the contribution of overland/sheetwash sediment to total sediment yield help adjust and calibrate this model. (Eq. :eq:`e` with sum calculated over the watershed area)
+        * **sed_export** (units: metric tons/watershed/year): Total amount of sediment exported to the stream per watershed. This should be compared to any observed sediment loading at the outlet of the watershed. Knowledge of the hydrologic regime in the watershed and the contribution of overland/sheetwash sediment to total sediment yield help adjust and calibrate this model. (Eq. :eq:`e` with sum calculated over the watershed area)
 
-        * **usle_tot** (units: tons/watershed/year): Total amount of potential soil loss in each watershed calculated by the USLE equation. (Sum of USLE from :eq:`usle` over the watershed area)
+        * **usle_tot** (units: metric tons/watershed/year): Total amount of potential soil loss in each watershed calculated by the USLE equation. (Sum of USLE from :eq:`usle` over the watershed area)
 
-        * **avoid_exp** (units: tons/watershed/year): The sum of avoided export in the watershed. (Sum of :math:`AEX_i` from :eq:`aex_i` over the watershed area)
+        * **avoid_exp** (units: metric tons/watershed/year): The sum of avoided export in the watershed. (Sum of :math:`AEX_i` from :eq:`aex_i` over the watershed area)

```

Comment:
```suggestion
        * **avoid_exp** (units: t/(watershed*year)): The sum of avoided export in the watershed. (Sum of :math:`AEX_i` from :eq:`aex_i` over the watershed area)

```

Reply:
My usual inclination is to spell things out as much as possible, and consistently. So personally I'd be more in favor of "metric tons/(watershed*year)" (or "metric tons/watershed/year", which reads a little more clearly to me, but let's pick one way of writing it and do it consistently throughout the UG). If I was a beginner with the models, I'm not sure I'd know what "t" stands for.